src/Simulation/Simulation.jl

```diff
@@ -930,7 +910,7 @@ function _calculate_potential!( sim::Simulation{T, CS}, potential_type::UnionAll
                                                 device_array_type = device_array_type,
                                                 not_only_paint_contacts = not_only_paint_contacts, 
                                                 paint_contacts = paint_contacts,
-                                                sor_consts = sor_consts )
+                                                sor_consts = is_last_ref ? T(1) : sor_consts )
```

Comment:
Out of curiosity, is there a difference in performance between `T(0)` and `zero(T)` (or between `T(1)` and `one(T)`)?
And should we come up with a consistent way of defining a zero or a one in SSD, as we use both ways in different parts of the code?

Reply:
I don't think it does. Did a simple test. 
I guess we should at some point. PR is welcome.

Reply:
@oschulz